### ISSUE TITLE
Chinese entity can not be extracted

### ISSUE THREAD

**Issue by saxh** [botfront/botfront#614]
**Describe the bug**
![Xnip2020-05-26_09-56-37](https://user-images.githubusercontent.com/22788088/82853120-5f5a6a00-9f37-11ea-9ea8-16244b2dd8d9.jpg)
![Xnip2020-05-26_09-56-07](https://user-images.githubusercontent.com/22788088/82853127-65e8e180-9f37-11ea-8f25-b7830cb59271.jpg)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by znat** [comment]
From your screenshot it seems some passwords get extracted but not all of them. Correct?

**Comment by saxh** [comment]
> From your screenshot it seems some passwords get extracted but not all of them. Correct?

No,it was that the "password" in English can be extracted while the "password" in Chinese can't when I underline the word.

**Comment by znat** [comment]
What is your nlu pipeline? Which tokenizer are you using?

**Comment by saxh** [comment]
> What is your nlu pipeline? Which tokenizer are you using?

I tried with jieba, whitespace and No tokenizer for bert.

**Comment by yingkai1181** [comment]
I have exact same problem ^^, is this being fixed?~

thanks

**Comment by aschao** [comment]
you can select Enable ono-whitespace language mode
